app/controllers/sprangular/locale_controller.rb

```diff
@@ -0,0 +1,14 @@
+module Sprangular
+  ##
+  # API Endpoint to set locale for session
+  class LocaleController < Sprangular::BaseController
+    def set
```

Comment:
actually, scratch that, forgot about our discussion earlier about reloading the whole site when locale changes

should be an http `GET` instead:

in config/routes.rb we can use `resources :locale, only: :show`

and the url would be `/locale/en` to do the redirection